pkgs/tools/misc/youtube-dl/default.nix

```diff
@@ -10,33 +9,36 @@ stdenv.mkDerivation rec {
     sha256 = "1crfada7vq3d24062wr06sfam66cf14j06wnhg7w5ljzrbynvpll";
   };
 
-  buildInputs = [ python makeWrapper zip pandoc ];
+  nativeBuildInputs = [ pandoc ];
+
+  buildInputs = [ python makeWrapper zip ];
 
   patchPhase = ''
-    rm youtube-dl
+    rm ./youtube-dl
   '';
 
-  configurePhase = ''
-    makeFlagsArray=( PREFIX=$out SYSCONFDIR=$out/etc PYTHON=${python}/bin/python )
-  '';
+  makeFlags = [
+    "PREFIX=$(out)"
+    "SYSCONFDIR=$(out)/etc"
+    "PYTHON=${python.interpreter}"
+  ];
 
   postInstall = ''
-    # ffmpeg is used for post-processing and fixups
     wrapProgram $out/bin/youtube-dl --prefix PATH : "${ffmpeg}/bin"
   '';
 
-  meta = {
-    homepage = "http://rg3.github.com/youtube-dl/";
-    repositories.git = https://github.com/rg3/youtube-dl.git;
```

Comment:
On Thursday, May 28, 2015 06:55:06 Cody Opel wrote:
> >      wrapProgram $out/bin/youtube-dl --prefix PATH : "${ffmpeg}/bin"
> >    
> >    '';
> > 
> > -  meta = {
> > -    homepage = "http://rg3.github.com/youtube-dl/";
> > -    repositories.git = https://github.com/rg3/youtube-dl.git;
> 
> I see your point, but wouldn't the monitor be able to compare the version to
> Arch & Gentoo's repos allowing it to get the same information without
> needing this, this would just be an additional redundant source.  Also is
> the monitor even running anymore, it hasn't updated in over a month now.

Monitor is running again. Having multiple update sources is actually very good 
because update souces fail from time to time. Having the git repo url handy is 
actually a useful just like description, despite it being easily googleable.